<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetRadios">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetFavorites">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAddToFavorites">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Favorites</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRemoveFromFavorites">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>160</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove from Favorites</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>200</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonPlay</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buttonPlay_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buttonStop_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddToFavorites</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buttonAddToFavorites_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveFromFavorites</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buttonRemoveFromFavorites_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buttonSearch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_buttonPlay_clicked()</slot>
  <slot>on_buttonStop_clicked()</slot>
  <slot>on_buttonAddToFavorites_clicked()</slot>
  <slot>on_buttonRemoveFromFavorites_clicked()</slot>
  <slot>on_buttonSearch_clicked()</slot>
 </slots>
</ui>
